aderyn_core/src/detect/test_utils/load_contract.rs

```diff
@@ -0,0 +1,44 @@
+use crate::context::workspace_context::WorkspaceContext;
+use crate::framework::foundry::read_foundry_output_file;
+use crate::read_file_to_string;
+use crate::visitor::ast_visitor::Node;
+use std::path::PathBuf;
+
+pub fn load_contract(filepath: &str) -> WorkspaceContext {
```

Comment:
I think we can remove this file altogether. The only files using this function are _template.rs!